test/erlcloud_aws_tests.erl

```diff
@@ -156,6 +157,29 @@ get_service_status_test(_) ->
      ?_assertEqual(OKStatus, ok)
      ].
 
+
+%% Tests that if application environment variable "erlcloud.aws_config" contains
+%% overrides for default `#aws_config{}', those overrides are indeed applied.
+default_config_override_test(_) ->
```

Comment:
Please add one more test to cover `application:get_env(erlcloud, aws_config) = undefined` case